packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -159,13 +178,18 @@ const ProgressIndicatorItemTimeline = () => (
 
 const ProgressIndicatorItemIcon = ({
 	status,
+	isActive,
 }: {
 	status: ProgressIndicatorItemStatus;
+	isActive: boolean;
 }) => {
 	const { icon: Icon, iconColor } = statusMap[status];
 	const ringWidth = tokens.borderWidth.md;
 	const ringGap = 3;
 	const ringInset = ringWidth + ringGap;
+	const processedIconColor: IconColour =
```

Comment:
This doesn't match the design, because Cannot start yet should also have the border colour. However, I'd question the design and think that none of the icons should change border colour, since it's not applicable to all, and we already have the outline

Reply:
I discussed this with the others. I need to restrict the active state to only apply to the doing, started, and blocked statuses.

Reply:
Updated. The icon now changes to blue when active. None of the alternate coloured icons are allowed to be set to active. The sytem will error if they are set to active.